pkg/virt-handler/options_test.go

```diff
@@ -345,4 +345,40 @@ var _ = Describe("Parsing VMI Options", func() {
 			Expect(actualTopology).To(Equal(expectedTopology))
 		})
 	})
+	It("should handle and avoid panic on host NUMA cells with nil values", func() {
```

Comment:
The DescriptionTable from before was looking nice. Were you having an issue with it?

Still, a unit test to validate that (1) this does not panic and (2) with know the expected value for now is enough I think.